internal/testhelper/certificatetest.go

```diff
@@ -0,0 +1,147 @@
+// Package testhelper implements utility routines required for writing unit tests.
+// The testhelper should only be used in unit tests.
+package testhelper
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"math/big"
+	"time"
+)
+
+var (
+	rsaRoot     CertTuple
+	rsaLeaf     CertTuple
+	ecdsaRoot   ECCertTuple
+	ecdsaLeaf   ECCertTuple
+	unsupported CertTuple
+)
+
+type CertTuple struct {
+	Cert       *x509.Certificate
+	PrivateKey *rsa.PrivateKey
+}
+
+type ECCertTuple struct {
+	Cert       *x509.Certificate
+	PrivateKey *ecdsa.PrivateKey
+}
+
+// init runs before any other part of this package.
+func init() {
+	setupCertificates()
+}
+
+// GetRSARootCertificate returns root certificate signed using RSA algorithm
+func GetRSARootCertificate() CertTuple {
+	return rsaRoot
+}
+
+// GetRSALeafCertificate returns leaf certificate signed using RSA algorithm
+func GetRSALeafCertificate() CertTuple {
+	return rsaLeaf
+}
+
+// GetECRootCertificate returns root certificate signed using EC algorithm
+func GetECRootCertificate() ECCertTuple {
+	return ecdsaRoot
+}
+
+// GetECLeafCertificate returns leaf certificate signed using EC algorithm
+func GetECLeafCertificate() ECCertTuple {
+	return ecdsaLeaf
+}
+
+// GetUnsupportedCertificate returns certificate signed using RSA algorithm with key size of 1024 bits
+// which is not supported by notary.
+func GetUnsupportedCertificate() CertTuple {
+	return unsupported
+}
```

Comment:
Having global gives less control than the function, because anyone can read or change the global variable at any time. Also, having function will allow us to change variable name/representation internally without affecting callers.